### PR TITLE
VACMS 18711 - vet center banner image

### PR DESCRIPTION
**Pull request by eselkin**
## Summary

- Adds media query to content-build for Banner image media for Vet Center. Currently this is the only media query outside of a node in the platform so it needs to get added into the node in metalsmith.
- Sitewide team

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18711

## Testing done

- Manually tested on Vet Center pages

## Screenshots

<details>
<summary>Old Vet Center Desktop</summary>

</details>

<details>
<summary>New Vet Center Desktop</summary>

</details>

<details>
<summary>New Vet Center Desktop showing padding</summary>

</details>

<details>
<summary>New Vet Center mobile</summary>

</details>

## What areas of the site does it impact?

Vet Center pages

## Acceptance criteria

- [x] Create a GraphQL query to fetch the Vet Center banner image from the centralized content in Drupal. (No longer applicable)
- [x] Banner image is presented on Vet Center pages
- [x] Implement responsive breakpoints for the banner image (not using breakpoints, handled by img component controls of DST stylesheets)
- [x] Follow all annotations in linked Figma designs

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user (unauthenticated)

## Requested Feedback

Check RI or tugboat
https://web-codylbelkq0gdivw6x7tykigmeqloewe.demo.cms.va.gov/boston-vet-center/

### CODE REVIEW

**Comment by eselkin** [comment]
Seems debatable whether we need a `picture > source > srcset` instead of just an img tag, but since the original image is so small, it's likely just the one is good, unless we really need to conserve even more on mobile. It's just that a reduced-pixel image (400 instead of 1000) is still 47kb, so you only save 13 kb.

File: src/site/stages/update-vet-center-pages.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const { logDrupal: log } = require('./build/drupal/utilities-drupal');
+
+function updateVetCenterPages(drupalData) {
+  const mediaQuery = drupalData.data.mediaQuery.entities[0];
+  for (let i = 0; i < drupalData.data.nodeQuery.entities.length; i += 1) {
+    if (drupalData.data.nodeQuery.entities[i].entityBundle === 'vet_center') {
+      // Insert the media query into the Vet Center object
+      // We have to do this because we cannot search for the
+      // media query inside the Vet Center graphQL query
+      // eslint-disable-next-line no-param-reassign
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/stages/update-vet-center-pages.js]: Removed

**Comment by eselkin** [comment]
@laflannery changes up on tugboat

**Comment by laflannery** [comment]
Looks good, approved!

**Review by randimays — APPROVED** [review]
LGTM!

**Comment by eselkin** [comment]
Re-running because something errored and it wouldn't just let me re-run failed

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**